lib/support/clone_vm.rb

```diff
@@ -31,16 +31,18 @@ def initialize(conn_opts, options)
     end
 
     def clone
-
       # set the datacenter name
       dc = vim.serviceInstance.find_datacenter(options[:datacenter])
       src_vm = dc.find_vm(options[:template])
-      hosts = dc.hostFolder.children
+
+      raise format("Unable to find template: %s", options[:template]) if src_vm.nil?
```

Comment:
Just to verify, other then template, maybe we should add other options here? `resource_pool`? `host`? etc etc?